apps/angular/src/app/features/anime/anime-details-page/anime-details-page.component.html

```diff
@@ -0,0 +1,96 @@
+<ng-container *ngIf="details$ | async as details">
+	<div class="anime-header">
+		<div class="anime-titles">
+			<h1 class="typography-title anime-title" *ngIf="details.titleEnglish.length > 0; else EmptyEnglishTitle">{{details.titleEnglish}}</h1>
+			<h3 class="typography-subtitle anime-subtitle" *ngIf="details.titleJapanese.length > 0; else EmptyJapaneseTitle">{{details.titleJapanese}}</h3>
```

Comment:
You have the option not to create an empty template for each case with the same text
or you can create pipe

```suggestion
			<h1 class="typography-title anime-title">{{ details.titleEnglish.length > 0 ? details.titleEnglish : 'unknown' }}</h1>
```